smart_mediation/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.hmscl.huawei.smart_mediation">
+
+    <application
```

Comment:
is it necessary?

Reply:
application tag is removed from module's manifest, it is not necessary